app/src/main/graphql/post.graphql

```diff
@@ -0,0 +1,7 @@
+query GetProjectUpdateComments($id: ID!,$limit: Int!, $cursor: String){
```

Comment:
Could we move this query to project.graphql?

Reply:
Done